assets/js/utilities/button.js

```diff
@@ -0,0 +1,27 @@
+// AngularJS Version
+// This creates a directive that uses the same data-toggle attribute, so the html is the same
```

Comment:
Yeah. This was worded rather ambiguously. It is the AngularJS implementation of the jQuery powered Bootstrap Button. The second line means that we are handling the data-toggle attribute manually in the javascript so that it behaves as if the Bootstrap jQuery file was the one handling changes to the element (but we don't want to have to load in jQuery because it's big).

Reply:
Ah brilliant--can we update the comment to be slightly more clear? I think the reasoning of not having to load in jQuery is particularly important